tests/unit/fixtures/datasets/open_images_v4/classification/test-annotations-human-imagelabels.csv

```diff
@@ -0,0 +1,20 @@
+ImageID,Source,LabelName,Confidence
+000026e7ee790996,verification,/m/04hgtk,0

```

Comment:
@lm-lily Sorry again.
In my opnion, copy binary image `000026e7ee790996.jpg` is not good idea.
Is is impossible to use symlink or more small size dummy image?
681KBx2 is not reasonable for dummy image size.

Reply:
@iizukak 
Yes, actually I am planning to remove `open_image_v4` as classification dataset.
I would like to use the originally available `dummy_classification` dataset to replace it.
If it is ok, I will commit the changes.

Reply:
@lm-lily Year, good to use `dummy_classification `